docs/reference/cli/completion.md

```diff
@@ -115,5 +115,5 @@ steampipe completion zsh
 
 Enable auto-complete for every new session (execute once).  You will need to start a new shell for this setup to take effect:
 ```bash
-steampipe completion zsh > "${fpath[1]}/steampipe"
+$ steampipe completion zsh > "${fpath[1]}/_steampipe" && compinit
```

Comment:
@binaek I am not familiar with zsh, so I'll have to assume this is correct.  I see that the line starts with `$` though - is that correct, or is that the prompt?  We don't usually include the prompt because the examples should work if you just cut and paste them...

Reply:
Ah! I missed that - let me push a fix